code/src/components/ThoughtsForm.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const ThoughtsForm = ({ newThought, onNewThoughtChange, onFormSubmit }) => {
+  return (
+    <div className="form-container">
+      <form onSubmit={onFormSubmit}>
+        <h1>What is making you happy right now?</h1>
+        <textarea
+          value={newThought}
+          onChange={onNewThoughtChange}
+          placeholder="Type your happy thoughts here..."
+          maxLength="140" />
+        <div className="counter">{0 + newThought.length} / {140 - newThought.length} </div>
```

Comment:
Nice way of making a counter. Is it ok if I borrow this idea?

Reply:
Of course! I borrowed it too from somewhere. :)